### PR TITLE
More PCH improvements (MSVC) 

### PR DESCRIPTION
**Pull request by SuperV1234**

### CODE REVIEW

File: Source/AliveLibAE/CMakeLists.txt [hunk 1]
```diff
@@ -415,11 +413,7 @@ target_compile_features(AliveLibAE
     PRIVATE cxx_variadic_templates)
 
 if (MINGW)
-    if (CMAKE_CXX_COMPILER_ID STREQUAL "Clang")
-        set(WIN32_LIBS "imm32")
-    else()
-        set(WIN32_LIBS "")
-    endif()
+    set(WIN32_LIBS "imm32")
```
Comment (SuperV1234) [inline Source/AliveLibAE/CMakeLists.txt:416]: Turns out this is needed for both gcc and clang when doing PCH builds, dunno but it works.

Comment (paulsapps) [reply Source/AliveLibAE/CMakeLists.txt:416]: strange how the hosted CI is working without too

Comment (SuperV1234) [reply Source/AliveLibAE/CMakeLists.txt:416]: does the hosted CI even build with MinGW?

Comment (paulsapps) [reply Source/AliveLibAE/CMakeLists.txt:416]: nope its MSVC, clang and gcc (so I would have expected the clang build to fail?)

Comment (SuperV1234) [reply Source/AliveLibAE/CMakeLists.txt:416]: clang/gcc need imm32 only for MinGW

**Pull request merged**